models/attachment.go

```diff
@@ -79,7 +79,11 @@ func (a *Attachment) LinkedRepository() (*Repository, UnitType, error) {
 			return nil, UnitTypeIssues, err
```

Comment:
Maybe the correct unitType (UnitTypePullRequests instead of UnitTypeIssues) is of interest here, too?!

Reply:
don't have an issue to be able to check if it is a pull

Reply:
and in any case if err is not nil then the unitType is not used: ```go
repository, unitType, err := attach.LinkedRepository()
	if err != nil {
		ctx.ServerError("LinkedRepository", err)
		return
	}
```

Reply:
Oh, of course, you're right.